content/docs/use-cases/versioning-data-and-model-files/tutorial.md

```diff
@@ -354,14 +354,14 @@ a monolithic way. It uses the `save_bottleneck_feature` function to
 pre-calculate the bottom, "frozen" part of the net every time it is run.
 Features are written into files. The intention was probably that the
 `save_bottleneck_feature` can be commented out after the first run, but it's not
-very convenient having to remember to do si it every time the dataset changes.
-
-Here's where the [pipelines](/doc/command-reference/pipeline) feature of DVC
-comes in handy. We touched on it briefly when we described `dvc run` and
-`dvc repro`. The next step would be splitting the script into two parts and
-utilizing pipelines. See [this example](/doc/tutorials/pipelines) to get
-hands-on experience with pipelines, and try to apply it here. Don't hesitate to
-join our [community](/chat) and ask any questions!
+very convenient having to remember to do so every time the dataset changes.
+
+Here's where the [pipelines](/doc/command-reference/dag) feature of DVC comes in
+handy. We touched on it briefly when we described `dvc run` and `dvc repro`. The
+next step would be splitting the script into two parts and utilizing pipelines.
+See [this example](/doc/start/data-pipelines) to get hands-on experience with
```

Comment:
/doc/start/data-pipelines is not an example. It's an entire guide on pipelines. Let's do
```suggestion
See [Data Pipelines](/doc/start/data-pipelines) to get hands-on experience with
```